src/components/Card.svelte

```diff
@@ -71,7 +74,7 @@
                 target="_blank"
                 rel="noopener noreferrer"
               >
-                See more
+                 {ButtoDisplayoption}
```

Comment:
why enum is put into dom element

Reply:
that is why it become [object Object]

Reply:
oh, because I thought it would work like other places.

Reply:
That's what happened when you use the same name to create props & enums (by practice props should not use TitleCase)